### ISSUE TITLE
minpac#update() results in "list index out of range: 0"

### ISSUE THREAD

**Issue by bstaletic** [k-takata/minpac#32]
Sometimes, when I run `:call minpac#update()` I get the following error:
```
  Error detected while processing function minpac#update[2]..minpac#impl#update[35]..<SNR>76_update_single_plugin[34]..<SNR>76_start_job[6]..<lambda>6[1]..<SNR>77_exit_cb[2].  .<
  SNR>76_job_exit_cb[12]..<SNR>76_get_plugin_revision[3]..<SNR>76_system[10]..<lambda>12[1]..<SNR>77_exit_cb[2]..<SNR>76_job_exit_cb[12]..<SNR>76_get_plugin_revision[3]..<SNR  >7
  6_system[10]..<lambda>18[1]..<SNR>77_exit_cb[2]..<SNR>76_job_exit_cb[12]..<SNR>76_get_plugin_revision[3]..<SNR>76_system[10]..<lambda>24[1]..<SNR>77_exit_cb[2]..<SNR>76_job  _e
  xit_cb[12]..<SNR>76_get_plugin_revision[3]..<SNR>76_system[10]..<lambda>42[1]..<SNR>77_exit_cb[2]..<SNR>76_job_exit_cb[12]..<SNR>76_get_plugin_revision[3]..<SNR>76_system[1  0]
  ..<lambda>48[1]..<SNR>77_exit_cb[2]..<SNR>76_job_exit_cb[12]..<SNR>76_get_plugin_revision[3]..<SNR>76_system[10]..<lambda>36[1]..<SNR>77_exit_cb[2]..<SNR>76_job_exit_cb[12]  ..
  <SNR>76_get_plugin_revision:
  line    5:
  E684: list index out of range: 0
  E15: Invalid expression: l:res[1][0]
```

I'm unfortunately unable to get a reproducible case. But it seems to occur more frequently when there is a big update of a single plugin. Though this time it happened it was when I was sure there are no updates available.

My vimrc:

```viml
  " Minpac package manager
  packadd minpac
  call minpac#init()
  call minpac#add('k-takata/minpac', {'type': 'opt'})

  " Mandatory plugins
  call minpac#add('airblade/vim-gitgutter')
  call minpac#add('christoomey/vim-tmux-navigator')
  call minpac#add('easymotion/vim-easymotion')
  call minpac#add('honza/vim-snippets')
  call minpac#add('idanarye/vim-vebugger')
  call minpac#add('jiangmiao/auto-pairs')
  call minpac#add('junegunn/vim-slash')
  call minpac#add('kshenoy/vim-signature')
  call minpac#add('Lokaltog/vim-distinguished')
  call minpac#add('mbbill/undotree')
  call minpac#add('neomake/neomake')
  call minpac#add('Shougo/vimproc.vim', {'do': 'silent! !make'})
  call minpac#add('simeji/winresizer')
  call minpac#add('SirVer/ultisnips')
  call minpac#add('tpope/vim-abolish')
  call minpac#add('tpope/vim-fugitive')
  call minpac#add('tpope/vim-repeat')
  call minpac#add('tpope/vim-surround')
  call minpac#add('tpope/vim-unimpaired')
  call minpac#add('Valloric/YouCompleteMe')
  call minpac#add('wellle/targets.vim')
  call minpac#add('weynhamz/vim-plugin-minibufexpl')
  call minpac#add('lambdalisue/vim-manpager')

  " Optional plugins
  call minpac#add('idanarye/vim-vebugger', {'type': 'opt'})
  call minpac#add('vim-scripts/a.vim', {'type': 'opt'})
```

My vim is 8.0.0628, but it also happened on 8.0.709.

**Comment by k-takata** [comment]
It seems that Vim sometimes fails to retrieve the output of `git rev-parse`.
I'm trying to fix it in #34.

**Comment by k-takata** [comment]
Already fixed in #34. Closing.
